compose.yaml

```
@@ -18,6 +18,21 @@ services:
       context: ./frontend/
     ports:
       - 5001:8080
+  mongo:
+    build:
+      context: ./database/
+    ports:
+      - 5002:27017
```

```suggestion
      - "${MONGO_PORT}:27017"
```

ports should generally not be hardcoded

---

also relevant for the other services, frontend & backend

---

Done